app/views/components/_table.html.erb

```diff
@@ -39,7 +39,7 @@
     <%= t.body do %>
       <% rows.each_with_index do |row, rowindex| %>
         <%= t.row do %>
-          <% row.each_with_index do |cell, cellindex| %>
+          <% row&.each_with_index do |cell, cellindex| %>
```

Comment:
I might have missed something, but would `row` ever be `nil` here if we're looping over a thing?

Reply:
I actually left this in by accident, before it occurred to me to use `compact` on the loop in `_event_log_table.rb`